android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.novoda.android">
+
+    <application android:allowBackup="true"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
```

Comment:
good question. no any downstream manifest has priority so they can set it to false